src/Views/CategoryView/categoryView.h

```diff
@@ -43,10 +45,14 @@ class CategoryView: public View {
 	Texture img_catbar;
 	Texture img_catbar_highlight;
 	Texture img_catbar_sep;
-	
+	Texture img_magnifying_glass;
+
 	unsigned int selectedCat;
 	std::vector<CategoryTab> categoryTabs;
 
 	int touchToCat(const Input &input);
+
+	IMEViewResult _ime_search_view_result = {};
```

Comment:
`CategoryView` should not know about this, it's only the concern of IME's `ListView`.

Reply:
Same as https://github.com/devnoname120/vhbb/pull/58#discussion_r253960117